packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.html

```diff
@@ -0,0 +1,79 @@
+<template>
+  <c-quantic-search-interface engine-id={engineId} skip-first-search>
+    <div class="slds-form-element">
+      <div class="slds-form-element__control">
+        <div class="slds-combobox_container">
+          <div
+            class="slds-combobox slds-dropdown-trigger slds-dropdown-trigger_click"
+            aria-expanded="true"
+            aria-haspopup="listbox"
+            role="combobox"
+          >
+            <div class="searchbox__container">
+              <div class={searchBoxContainerClass} role="none">
+                <template if:true={withoutSubmitButton}>
+                  <lightning-icon
+                    class="slds-icon slds-input__icon slds-input__icon_left"
+                    icon-name="utility:search"
+                    size="x-small"
+                    title=""
+                  ></lightning-icon>
+                </template>
+                <input
+                  id="combobox-with-button-id"
+                  class="slds-input slds-combobox__input searchbox__input"
+                  placeholder={placeholder}
+                  type="search"
+                  onkeyup={onKeyup}
+                  onfocus={onFocus}
+                  onblur={onBlur}
+                  aria-autocomplete="list"
+                  aria-controls="listbox-id"
+                  autocomplete="off"
+                  role="textbox"
+                />
+                <template if:false={isQueryEmpty}>
+                  <lightning-button-icon
+                    onclick={clearInput}
+                    icon-name="utility:close"
+                    variant="bare"
+                    class="slds-input__icon slds-input__icon_right input__icon_cursor"
+                    size="medium"
+                    name="Clear"
+                    title="Clear"
+                    aria-label="Clear"
+                    alternative-text="Clear"
+                  ></lightning-button-icon>
+                </template>
+              </div>
+              <template if:false={withoutSubmitButton}>
+                <button
```

Comment:
can we use lightning-button instead?

Reply:
I'm not sure because I didn't make this initially. But it's possible the lightning-buttons come with their own styling we'd have to fight against. Not sure though

Reply:
I didn't try for the searchbox because we have more styling I think than on the clear button.

We could look at it in the future I guess?

Reply:
![image](https://user-images.githubusercontent.com/2488155/130091780-7a9dfc66-4007-4e43-ab48-ad8ea5dd99eb.png)
Just tried it, will update the PR